workspace-manager-clienttests/render-config.sh

```diff
@@ -7,16 +7,21 @@ USER_DELEGATED_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/firecloud/dev/common/firec
 USER_DELEGATED_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/rendered/user-delegated-service-account.json
 TESTRUNNER_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/common/testrunner-sa
 TESTRUNNER_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/rendered/testrunner-service-account.json
+TESTRUNNER_IAM_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/integration/ichang/testrunner-sa
+TESTRUNNER_IAM_SERVICE_ACCOUNT_OUTPUT_PATH=$(dirname "$0")/rendered/testrunner-iam-service-account.json
```

Comment:
I removed `TESTRUNNER_SERVICE_ACCOUNT_VAULT_PATH` and `TESTRUNNER_SERVICE_ACCOUNT_OUTPUT_PATH`. The `IAM` is a better description of these accounts to distinguish them from the K8s SAs.

Reply:
Differentiating the GCP service account from the K8S/GKE one makes sense to me. But all the other paths in this file are also fetching GCP service accounts right? So I'm confused why the test runner one has the IAM prefix but the others don't. Perhaps we can include K8S or GKE in the name of the Kuberenetes type of SA, and keep referring to the GCP variety as just SA?

Reply:
To better organize the GCP versus k8s service accounts. I moved the GCP test runner service accounts in the vault one level up and placed it in the `common` path rather than inside a path with a `namespace` since the GCP service account works at the cluster level (it has Kubernetes Viewer role). So now the convention should be much cleaner.